lib/full-site-editing/page-templates.php

```diff
@@ -17,17 +17,12 @@ function gutenberg_load_block_page_templates( $templates, $theme, $post ) {
 	if ( ! gutenberg_is_fse_theme() ) {
 		return $templates;
 	}
-	$config_file = locate_template( 'experimental-theme.json' );
-	if ( ! file_exists( $config_file ) ) {
-		return $templates;
-	}
-	$data           = json_decode(
-		file_get_contents( $config_file ),
-		true
-	);
+
+	$resolver       = new WP_Theme_JSON_Resolver();
+	$data           = $resolver->get_theme_data()->get_page_templates();
```

Comment:
Circling back to this.  Now that `theme` is static.  Do we need to call `new` here?  Or would something like:
`WP_Theme_JSON_Resolver::get_theme_data()->get_page_templates();` work?

Reply:
Here's the follow-up that bundles non-behavioral changes: https://github.com/WordPress/gutenberg/pull/28855 (it also makes `get_theme_data` static).